sdk/spring/azure-spring-boot/src/main/java/com/microsoft/azure/keyvault/spring/KeyVaultEnvironmentPostProcessorHelper.java

```diff
@@ -133,6 +136,8 @@ public TokenCredential getCredentials(String normalizedName) {
         final String tenantId = getPropertyValue(normalizedName, Property.TENANT_ID);
         final String certificatePath = getPropertyValue(normalizedName, Property.CERTIFICATE_PATH);
         final String certificatePassword = getPropertyValue(normalizedName, Property.CERTIFICATE_PASSWORD);
+        final String authorityHost = getPropertyValue(normalizedName, Property.AUTHORITY_HOST) == null ?
+            new IdentityClientOptions().getAuthorityHost() : getPropertyValue(normalizedName, Property.AUTHORITY_HOST);
```

Comment:
Is this the default value for authority host?
`new IdentityClientOptions().getAuthorityHost()`

Reply:
yes,the constructor of IdentityClientOptions will initialize authorityHost  with default value